telegram/_bot.py

```diff
@@ -6907,14 +6940,14 @@ async def get_forum_topic_icon_stickers(
         connect_timeout: ODVInput[float] = DEFAULT_NONE,
         pool_timeout: ODVInput[float] = DEFAULT_NONE,
         api_kwargs: JSONDict = None,
-    ) -> List[Sticker]:
+    ) -> Tuple[Sticker]:
```

Comment:
```suggestion
    ) -> Tuple[Sticker, ...]:
```
Doesn't mypy catch this?

Reply:
So far it doesn't because we have a `# type: ignore[return-value]` in place. This is necessary because `de_list` returns `Tuple[Optional[Thing], …]` instead of `Tuple[Thing, …]`.
I'm changing that + also adapting a bit of type hinting in order to drop most of the `type: ignore` comments in `_bot.py`. but since that may be viewed as a bit beyodn this PR, I'm doing that in a standalone commit, so we can cherry pick it into a different pr

Reply:
thumbs up means leave as is or cherry pick into a different pr? :D

Reply:
oh sorry, I didn't see a question, but I'm okay with it in this PR.